test/source/tests/compose.ts

```diff
@@ -634,6 +634,25 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       })
     );
 
+    test(
+      'compose - check apostrophe file name encoding',
+      testWithBrowser(async (t, browser) => {
```

Comment:
I tried to run this test in `master` branch and it still passes, so probably something is not correct in attachments check here

Reply:
Hmm... let me check

Reply:
Hmm.. I tried to run test in master branch and it fails. Could you try again?

Reply:
For `master` branch I mean taking `emailjs-mime-codec.js` from `master` branch, so it doesn't include fix.
I tried it again - still passes, can you try to run test this way?